stores/sql/metrics.go

```diff
@@ -0,0 +1,142 @@
+package sql
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/internal/sql"
+)
+
+const (
+	tableContractPruneMetric = "contract_prunes"
```

Comment:
Trying to avoid these kind of constants. They reduce the readability imo. with only minor upsides.